gnucash/import-export/aqb/gschemas/org.gnucash.GnuCash.dialogs.import.hbci.gschema.xml.in

```diff
@@ -27,8 +27,8 @@
     </key>
     <key name="verbose-debug" type="b">
       <default>false</default>
-      <summary>Verbose HBCI debug messages</summary>
-      <description>Enables verbose debug messages for HBCI/AqBanking Online Banking.</description>
+      <summary>Verbose FinTS debug messages</summary>
+      <description>Enables verbose debug messages for FinTS/AqBanking Online Banking.</description>
```

Comment:
"FinTS/AqBanking Online Banking" is rather ponderous and redundant. Since the user is already in a FinTS related dialog surely it's sufficient to say "Enable verbose debug messages."

Reply:
No, gschemas store preferences. But there it is also part of [tab Online Banking](https://www.gnucash.org/docs/v5/C/gnucash-manual/set-prefs.html#prefs-online). But I would keep AqBanking in the tooltip to avoid confusion with other log modules.

Reply:
@CWehli  it is the general AQBANKING_LOGLEVEL [wiki:AqBanking#Debugging](https://wiki.gnucash.org/wiki/AqBanking#Debugging) neither FinTS nor HBCI.

Reply:
Are the debug messages also written to the AqBanking log window?
If so, the tooltip in the preferences dialogue could be supplemented accordingly.

Reply:
That preference sets the AQBanking and Gwenhywfar logging levels to debug and info respectively unless they've been set with environment variables ([code](https://github.com/Gnucash/gnucash/blob/6d0007ec308e46ad0b0226ba909004dc2e8f5886/gnucash/import-export/aqb/gnc-ab-utils.c#L122)). As [explained in the wiki](https://wiki.gnucash.org/wiki/AqBanking#Debugging) the Gwenhyfar output goes to `stdout` and the AQBanking output goes to the tracefile *if* the user starts GnuCash with `--log aqbanking=debug`. I don't think that it affects what gets written to the log window.